colossalai/graph/passes/adding_split_node_pass.py

```diff
@@ -0,0 +1,54 @@
+import torch
+
+from torch.fx import symbolic_trace
+from torch.fx.node import Node
+from torch.fx.passes.split_module import split_module
```

Comment:
Can you do a torch version check when importing `torch.fx`. Colossal-AI supports minimum torch 1.8 and I think `torch.fx` does not exist in the older versions.